src/common/components/build-row/buildRow.css

```diff
@@ -0,0 +1,8 @@
+.commitIcon {
```

Comment:
In BSI we use CSS Modules. This makes CSS class names accessible in JS code.
Having them in camelCase makes it possible to use simple dot notation (as in `styles.commitIcon`) rather then accessing by string (`styles['commit-icon']`).

So yes, there is a reason for camelCase.

But I agree, that ideally we should consider moving to using same naming standards as Vanilla in BSI CSS.

Reply:
I think I prefer `styles['commit-icon']`. It makes it clearer that you're import CSS rather than some similar JavaScript concept.

Anyway, merge away.

Reply:
OK. From that point of view I agree on readiness of this approach.

We need to do it this way in modules imported from Vanilla anyway, so it makes sense to consider migrating into such naming in rest of the styles as well later on.